<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="nl-nl">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">e943d85b3e0050ac2ac9b0225303d067f2a1bf4e</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.AppDomainUnloadedException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">f2eeee9d955d5b92dcdd515d6b5ce0350b646cd0</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">96ab56d224f526a6f3808f93ed7265997110d6de</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when an attempt is made to access an unloaded application domain.</source>
          <target state="translated">De uitzondering die wordt gegenereerd wanneer wordt geprobeerd om een verwijderde toepassingsdomein.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>In the .NET Framework version 2.0, an AppDomainUnloadedException that is not handled in user code has the following effect:      -   If a thread was started in managed code, it is terminated.</source>
          <target state="translated">In het .NET Framework versie 2.0, een AppDomainUnloadedException dat niet is verwerkt in gebruikerscode heeft de volgende effect:-als een thread is gestart in beheerde code, wordt het beëindigd.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>The unhandled exception is not allowed to terminate the application.</source>
          <target state="translated">De niet-verwerkte uitzondering is niet toegestaan om de toepassing te beëindigen.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>-   If a task is executing on a &lt;xref:System.Threading.ThreadPool&gt; thread, it is terminated and the thread is returned to the thread pool.</source>
          <target state="translated">-Als een taak wordt uitgevoerd op een &lt;xref:System.Threading.ThreadPool&gt;thread, wordt het beëindigd en de thread wordt geretourneerd aan de threadgroep.&lt;/xref:System.Threading.ThreadPool&gt;</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT">
          <source>The unhandled exception is not allowed to terminate the application.</source>
          <target state="translated">De niet-verwerkte uitzondering is niet toegestaan om de toepassing te beëindigen.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT">
          <source>-   If a thread started in unmanaged code, such as the main application thread, it is terminated.</source>
          <target state="translated">-Als een thread is gestart in onbegeleide code, zoals de hoofdtoepassingsthread, wordt het beëindigd.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT">
          <source>The unhandled exception is allowed to proceed, and the operating system terminates the application.</source>
          <target state="translated">De niet-verwerkte uitzondering is toegestaan om door te gaan en de toepassing door het besturingssysteem wordt beëindigd.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>AppDomainUnloadedException uses the HRESULT COR_E_APPDOMAINUNLOADED, which has the value 0x80131014.</source>
          <target state="translated">AppDomainUnloadedException maakt gebruik van de HRESULT COR_E_APPDOMAINUNLOADED, dat de waarde 0x80131014 heeft.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT">
          <source>For a list of initial property values for an instance of AppDomainUnloadedException, see the &lt;xref:System.AppDomainUnloadedException.%23ctor%2A&gt; constructors.</source>
          <target state="translated">Zie voor een lijst met eerste eigenschapswaarden voor een exemplaar van AppDomainUnloadedException de &lt;xref:System.AppDomainUnloadedException.%23ctor%2A&gt;constructors.&lt;/xref:System.AppDomainUnloadedException.%23ctor%2A&gt;</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.AppDomainUnloadedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.AppDomainUnloadedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property of the new instance to a system-supplied message that describes the error, such as "The target application domain has been unloaded."</source>
          <target state="translated">Deze constructor initialiseert de &lt;xref:System.Exception.Message%2A&gt;eigenschap van het nieuwe exemplaar op een systeem geleverd bericht die de fout wordt beschreven, zoals "domein van de doeltoepassing is verwijderd."&lt;/xref:System.Exception.Message%2A&gt;</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">Dit bericht wordt rekening gehouden met de huidige systeemcultuur.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.AppDomainUnloadedException&gt;.</source>
          <target state="translated">De volgende tabel toont de eerste waarden voor een exemplaar van &lt;xref:System.AppDomainUnloadedException&gt;.&lt;/xref:System.AppDomainUnloadedException&gt;</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|</source>
          <target state="translated">| Eigenschap | Waarde |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| Een null-verwijzing (<ph id="ph1">`Nothing`</ph> in Visual Basic). |   | &lt;xref:System.Exception.Message%2A&gt;| De gelokaliseerde fouttekenreeks bericht. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.AppDomainUnloadedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.AppDomainUnloadedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met een opgegeven foutbericht.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT">
          <source>The content of the <ph id="ph1">`message`</ph> parameter is intended to be understood by humans.</source>
          <target state="translated">De inhoud van de <ph id="ph1">`message`</ph> parameter is bedoeld om te worden geïnterpreteerd door mensen.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">De aanroeper van deze constructor is vereist om ervoor te zorgen dat deze tekenreeks gelokaliseerd voor de huidige systeemcultuur.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.AppDomainUnloadedException&gt;.</source>
          <target state="translated">De volgende tabel toont de eerste waarden voor een exemplaar van &lt;xref:System.AppDomainUnloadedException&gt;.&lt;/xref:System.AppDomainUnloadedException&gt;</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Eigenschap | Waarde |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| Een null-verwijzing (<ph id="ph1">`Nothing`</ph> in Visual Basic). |   | &lt;xref:System.Exception.Message%2A&gt;| De fouttekenreeks bericht. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve">
          <source>The message that describes the error.</source>
          <target state="translated">Het bericht dat de fout wordt beschreven.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.AppDomainUnloadedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialized data.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.AppDomainUnloadedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met de geserialiseerde gegevens.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</source>
          <target state="translated">Deze constructor is aangeroepen tijdens de deserialisatie op het object exception verzonden via een stroom opnieuw samenstellen.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>.</source>
          <target state="translated">Zie voor meer informatie <bpt id="p1">[</bpt>XML en SOAP-serialisatie<ept id="p1">](~/add/includes/ajax-current-ext-md.md)</ept>.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">Het object dat de gegevens geserialiseerd object bevat.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve">
          <source>The contextual information about the source or destination.</source>
          <target state="translated">De contextuele informatie over de bron- of doelserver.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.AppDomainUnloadedException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.AppDomainUnloadedException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT">
          <source>The content of the <ph id="ph1">`message`</ph> parameter is intended to be understood by humans.</source>
          <target state="translated">De inhoud van de <ph id="ph1">`message`</ph> parameter is bedoeld om te worden geïnterpreteerd door mensen.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">De aanroeper van deze constructor is vereist om ervoor te zorgen dat deze tekenreeks gelokaliseerd voor de huidige systeemcultuur.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering moet een verwijzing bevatten naar de vorige uitzondering in de &lt;xref:System.Exception.InnerException%2A&gt;eigenschap.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or a null reference if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">De &lt;xref:System.Exception.InnerException%2A&gt;eigenschap dezelfde waarde die is doorgegeven aan de constructor of een null-verwijzing als de &lt;xref:System.Exception.InnerException%2A&gt;eigenschap levert niet de waarde van de binnenste uitzondering voor de constructor.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.AppDomainUnloadedException&gt;.</source>
          <target state="translated">De volgende tabel toont de eerste waarden voor een exemplaar van &lt;xref:System.AppDomainUnloadedException&gt;.&lt;/xref:System.AppDomainUnloadedException&gt;</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Eigenschap | Waarde |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| De verwijzing naar de interne uitzondering. |   | &lt;xref:System.Exception.Message%2A&gt;| De fouttekenreeks bericht. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve">
          <source>The message that describes the error.</source>
          <target state="translated">Het bericht dat de fout wordt beschreven.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">De uitzondering die de oorzaak van de huidige uitzondering.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`innerException`</ph> parameter is not a null reference, the current exception is raised in a <bpt id="p1">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">Als de <ph id="ph1">`innerException`</ph> parameter is niet een null-verwijzing, de huidige uitzondering is opgetreden tijdens een <bpt id="p1">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> blok die verantwoordelijk is voor de interne uitzondering.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>